.github/workflows/security-center-snippets.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
     - run: npm test
       env:
         GCLOUD_ORGANIZATION: 1081635000895
-        GOOGLE_SAMPLES_PROJECT: "long-door-651"
+        GOOGLE_SAMPLES_PROJECT: "project-a-id"
```

Comment:
issue: Use of this project is under discussion in #3830. It is probably not usable.

Reply:
Addressed.